.buildkite/premerge.steps.yaml

```diff
@@ -1,25 +1,15 @@
 ---
-ci_version: &ci_version "1.1"
-# This is designed to trap and retry failures because agent lost
-# connection. Agent exits with -1 in this case.
-agent_transients: &agent_transients
-  exit_status: -1
-  limit: 3
-
-# BK system error
-bk_system_error: &bk_system_error
-  exit_status: 255
-  limit: 3
-  
-# Job was interrupted by a signal (e.g. ctrl+c etc)
-bk_interrupted_by_signal: &bk_interrupted_by_signal
-  exit_status: 15
-  limit: 3
-
-# Hook failure
-bk_hook_failure: &bk_hook_failure
```

Comment:
How come we are removing this? This can still happen in the GDK and other Unreal parent repos before calling the CI repo

Reply:
It can happen in the EV-supplied initial script I suppose, however these yaml anchors will not affect it, since that script runs before this yaml is even read.